### ISSUE TITLE
error in I/O

### ISSUE THREAD

**Issue by diegofc00** [hgourvest/node-firebird#103]
In firebird 3.0

I got this
Error: I/O error during "CreateFile (open)" operation for file "DAUM.FDB", Error while trying to open file
    at doCallback (C:\sistemas\daum-api\node_modules\node-firebird\lib\index.js:1233:18)
    at C:\sistemas\daum-api\node_modules\node-firebird\lib\index.js:2928:21
    at C:\sistemas\daum-api\node_modules\node-firebird\lib\messages.js:151:25
    at search (C:\sistemas\daum-api\node_modules\node-firebird\lib\messages.js:117:13)
    at C:\sistemas\daum-api\node_modules\node-firebird\lib\messages.js:54:21
    at FSReqWrap.wrapper [as oncomplete] (fs.js:527:17)

Code:
var Firebird = require('node-firebird');
//var md5 = require('md5');
var options = {};
options.host = '127.0.0.1';
options.port = 3050;
options.database = 'DAUM.FDB';
options.user = 'sysdba';
options.password = 'masterkey';

var pool = Firebird.pool(10, options);
var k = 0;
pool.get(function (err, db) {
	if (err)
		throw err;

	db.sequentially('SELECT FIRST 500 SOME QUERY', function (row, index) {
		k = k + 1;
		console.log(k + ' => ' + md5(JSON.stringify(row)) + '\n');
	}, function (err) {
		db.detach();
	});
});

**Comment by diegofc00** [comment]
I'll fix.

**Comment by florjon** [comment]
how did you fix this ?

**Comment by HamdiBacem** [comment]
how did you fix this ?

**Comment by 1p0d** [comment]
Fixed this by moving the fdb file out of my projects directory and using an absolute path in `options.database`.
